clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3576,10 +3574,8 @@ void handleFailure(Throwable throwable) {
         return new ListPartitionReassignmentsResult(partitionReassignmentsFuture);
     }
 
-    private long calculateNextAllowedRetryMs(long currNextAllowedRetryMs) {
-        return (currNextAllowedRetryMs == 0) ?
-            time.milliseconds() + retryBackoffMs :
-            currNextAllowedRetryMs + retryBackoffMs;
+    private long calculateNextAllowedRetryMs(long now) {
```

Comment:
nit: As we always pass same value to this method, we can drop the parameter and use `time.milliseconds` within this method.

Reply:
Done.